poketex-git/PKGBUILD

```diff
@@ -37,6 +37,9 @@ check() {
 
 package() {
   cd "${pkgname%-git}"
+  mkdir -p "$pkgdir/usr/share/${pkgname%-git}/colorscripts/small/"
```

Comment:
Since you have `install -Dt` going on here anyway you don't need two `mkdir -p` calls.

Reply:
removed, Thanks!